csharp/extractor/Semmle.Extraction.CSharp/Entities/Constructor.cs

```diff
@@ -22,6 +32,12 @@ public override void Populate(TextWriter trapFile)
             trapFile.constructors(this, Symbol.ContainingType.Name, ContainingType, (Constructor)OriginalDefinition);
             trapFile.constructor_location(this, Location);
 
+            if (IsPrimary)
+            {
+                // Create a synthetic empty body for primary constructors.
+                Statements.SyntheticEmptyBlock.Create(Context, this, 0, Location);
```

Comment:
As discussed on Slack, this might also be useful for default constructors.

Reply:
Yes, that is a good point!
I would prefer to make furthermore improvements on follow up PRs. Is that ok?

Reply:
Yep, a separate PR sounds good.